wgetpaste

```diff
@@ -3,7 +3,7 @@
 # relying only on bash, sed, coreutils (mktemp/sort/tr/wc/whoami/tee) and wget
 # Copyright (c) 2007-2016 Bo Ørsted Andresen <bo.andresen@zlin.dk>
 
-VERSION="2.29"
+VERSION="2.30"
```

Comment:
Could you please not increase the version number here so I can get more things into the new release?